### PR TITLE
Add script to download IXI dataset

### PR DESCRIPTION
**Pull request by fepegar**
Usage:

```bash
$ ./download_ixi.sh
```

It takes an optional modality: T1, T2, PD, MRA... The default is T1.

### CODE REVIEW

**Comment by mathpluscode** [comment]
Feel free to merge by yourself? Or we want to do code review now already?

**Comment by fepegar** [comment]
Given the collaborative spirit of the project, I think it'd be nice to have at least one review from someone else.
Also, I'd always use "Squash and merge" (relevant when there are multiple commits in the PR).
What do you think?

**Comment by mathpluscode** [comment]
> Given the collaborative spirit of the project, I think it'd be nice to have at least one review from someone else.
> Also, I'd always use "Squash and merge" (relevant when there are multiple commits in the PR).
> What do you think?

Considering the nature of this repo, yes, squash should be good, will set it in the repo later.

File: download_ixi.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+MODALITY=${1:-T1}  # "T1" is the default value
```
Comment (mathpluscode) [inline download_ixi.sh]: maybe write down the possible values

Comment (fepegar) [reply download_ixi.sh]: Good point. Done.

**Review by mathpluscode — APPROVED** [review]

**Pull request merged**